drivers/include/shtc1.h

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2017, RWTH Aachen. All rights reserved.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_shtc1
+ * @ingroup     drivers_sensors
+ * @name        Device driver interface for the SHTC1 Temperature and humidity sensor
+ * @{
+ *
+ * @file
+ * @brief       Device driver interface for the SHTC1 Temperature and humidity sensor
+ *
+ * @author      Steffen Robertz <steffen.robertz@rwth-aachen.de>
+ * @author      Josua Arndt <jarndt@ias.rwth-aachen.de>
+ */
+
+#ifndef SHTC1_H
+#define SHTC1_H
+
+#include <stdint.h>
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum {
+    CRC_DISABLED = 0,
+    CRC_ENABLED
+} shtc1_crc_type_t;
+
+/**
+ * @brief   value struct, where all results obtained from the sensor will be saved
+ * @{
+ */
+typedef struct {
+    float temp;         /**< Temperature after an according call to the measurement function */
+    float rel_humidity; /**< Relative humidity after an according call to the measuerment funtion*/
+    unsigned int id;    /**< ID read from the sensor, only available after shtc1_id() was called */
+} shtc1_values_t;
+/** @} */
+
+/**
+ * @brief   settings struct with all relevant parameters
+ * @{
+ */
+typedef struct {
+    i2c_t bus;  /**< I2C bus descriptor */
+    uint8_t addr;   /**< I2C address of the sensor */
+    shtc1_crc_type_t crc;   /**< crc check enabled or disabled (CRC_ENABLED/CRC_DISABLED) */
+} shtc1_params_t;
+/** @} */
+
+/**
+ * @brief   device descriptor for the SHTC1
+ * @{
+ */
+typedef struct {
+    shtc1_values_t values;  /**< Values struct, where all read data will be stored */
```

Comment:
Just for my understanding: Why did you integrate the values struct here? For caching purposes?